targets/TARGET_STM/serial_api.c

```diff
@@ -690,4 +690,105 @@ int8_t get_uart_index(UARTName uart_name)
     return -1;
 }
 
+/*  Function to protect deep sleep while a seral Tx is ongoing on not complete
+ *  yet. Returns 1 if there is at least 1 serial instance with ongoing ransfer
+ *  0 otherwise.
+ */
+int serial_IsTxOngoing(void) {
```

Comment:
Shall this follow the style, `serial_is_tx_ongoing`  - assuming its in the HAL (I can see this is close to driver naming as `LL_USART_IsEnabled()`)

Reply:
You're right !  will do the change